src/main/java/org/christiangalsterer/stash/filehooks/plugin/hook/FileNameHook.java

```diff
@@ -122,4 +145,82 @@ public void validate(Settings settings, SettingsValidationErrors errors, Reposit
             }
         }
     }
+	
+	  /**
+     * Callback, collecting all the paths, changed in the requested change 
+     * range. 
+     */ 
+    private static class ChangesPathsCollector extends AbstractChangeCallback { 
+        private final Collection<String> changedPaths = new HashSet<String>(); 
+ 
+        @Override 
+        public boolean onChange(Change change) throws IOException {
+        	//isNotDeleteChange
+        	if ( change.getType() != ChangeType.DELETE )
+        	{
+        		changedPaths.add(change.getPath().toString());
+        	}
+            return true; 
+        } 
+ 
+        public Collection<String> getChangedPaths() { 
+            return changedPaths; 
+        } 
+ 
+    }  
+  			 	
+	 private String getPullRequestError(Collection<String> filteredFiles) { 
+	        final StringBuilder sb = new StringBuilder(); 
+	        sb.append(i18n.getText("filename-hook.error.pattern", "Pattern is not a valid regular expression")); 
```

Comment:
I think the error message is not correct as the message is about an invalid regular expression. Can you please double-check and fix?